plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergV2.java

```diff
@@ -169,8 +170,9 @@ public void testV2TableWithPositionDelete()
                 .withSpec(PartitionSpec.unpartitioned())
                 .buildPositionWriter();
 
+        PositionDelete<Record> positionDelete = PositionDelete.create();
         try (Closeable ignored = writer) {
-            writer.delete(dataFilePath, 0, GenericRecord.create(icebergTable.schema()));
+            writer.write(positionDelete.set(dataFilePath, 0, GenericRecord.create(icebergTable.schema())));
```

Comment:
Nit: It seems cleaner to set this before opening the writer
```java
PositionDelete<Record> positionDelete = PositionDelete.create()
        .set(dataFilePath, 0, GenericRecord.create(icebergTable.schema()));
try (Closeable ignored = writer) {
        writer.write(positionDelete);
}

Reply:
That is indeed much cleaner, thanks for noticing. I'll fix this once the release is done. I wanted to check if all the tests still pass, but ran into some issues running the tests locally.

Reply:
Done! 👍🏻